Amplify/Categories/DataStore/Model/Temporal/README.md

```diff
@@ -0,0 +1,58 @@
+## Custom Data Types
+
+`DataStore > Model > Temporal`
+
+Model-based programming aims to simplify data management on apps and enable developers to focus on the business and UI implementation while the technical aspects on persisting data are abstracted away from the core logic. Therefore, data types are a critical piece of it.
```

Comment:
seems like we want to say something like:
Model-based programming aims to simplify data management on apps by abstracting away the complexities of  data persistence from the core logic of the app.

Reply:
Cool, much simpler. Thanks!